src/d3d12/d3d12_renderer.cpp

```diff
@@ -462,23 +474,14 @@ namespace wr
 		{
 			LOGW("A render target has no transitions specified. Is this correct?");
 		}
-
-		d3d12::End(n_cmd_list);
 	}
 
 	void D3D12RenderSystem::StartComputeTask(CommandList * cmd_list, std::pair<RenderTarget*, RenderTargetProperties> render_target)
 	{

```

Comment:
functions StartComputeTask and StopComputeTask are now empty. concider using __declspec(deprecated("Not implemented")) or something similar